packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less

```diff
@@ -31,6 +31,11 @@
         .nui-step-header-ripple {
             pointer-events: none;
         }
+
+        &:focus {
+            .focus-outline();
+            border-radius: 5px;
```

Comment:
Is there a chance that we should use `@nui-radius-default` found [here](https://github.com/solarwinds/nova/blob/8928c3359be235534175f58815665301be0d4838/packages/bits/src/styles/nui-framework-variables.less#L71)? It has a value of 3px. If it needs to be 5px according to UX, maybe it would be good to put a variable to use in this file:

https://github.com/solarwinds/nova/blob/aba6f99353d6ce384cf8748c7f1fdb5a2e12c2b4/packages/bits/src/lib/wizard-v2/wizard-vars.less

Reply:
We can use @nui-radius-default here. This work in progress.